src/date-range-picker/__tests__/time-offset.test.ts

```diff
@@ -8,6 +8,30 @@ describe('Date range picker', () => {
   describe('time offset handling', () => {
     describe('setTimeOffset', () => {
       test.each([
+        [
```

Comment:
it looks like the tests are added for setTimeOffset but the change is made to shiftTimeOffset

Reply:
Removed these changes as it is no longer needed after this change https://github.com/cloudscape-design/components/pull/3111/files?w=1